modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaImpl.java

```diff
@@ -73,6 +74,10 @@ public class ReplicaImpl implements Replica {
 
     private final PlacementDriverMessageProcessor placementDriverMessageProcessor;
 
+    private final EventListener<PrimaryReplicaEventParameters> onReplicaElected = this::registerFailoverCallback;
```

Comment:
I may propose `onPrimaryReplicaElected` and `onPrimaryReplicaExpired`, but it's opinionated.

Reply:
Thanks! I will change the names.